test/functional/ui/messages2_spec.lua

```diff
@@ -177,14 +179,13 @@ describe('messages2', function()
       {16::}^                                                    |
     ]])
     -- Highlighter disabled when message is moved to cmdline #34884
-    feed('ls<CR>')
+    feed([[echo "bar\n"->repeat(&lines)<CR>]])
     screen:expect([[
       ^                                                     |
-      {1:~                                                    }|*8
+      {1:~                                                    }|*4
       {3:─────────────────────────────────────────────────────}|
       foo                                                  |
-                                                           |
-        1 %a   "[No Name]"                    line 1       |
+      bar                                                  |*6
```

Comment:
Last line should have a spill indicator if there more messages than were shown.

Reply:
Right this works with `cfg.msg.target == 'cmd'` but not `'msg'.